python/ray/serve/handle.py

```diff
@@ -63,13 +63,15 @@ class HandleOptions:
     multiplexed_model_id: str = ""
     stream: bool = False
     _router_cls: str = ""
+    request_protocol: RequestProtocol = RequestProtocol.UNDEFINED
 
     def copy_and_update(
         self,
         method_name: Union[str, DEFAULT] = DEFAULT.VALUE,
         multiplexed_model_id: Union[str, DEFAULT] = DEFAULT.VALUE,
         stream: Union[bool, DEFAULT] = DEFAULT.VALUE,
         _router_cls: Union[str, DEFAULT] = DEFAULT.VALUE,
+        request_protocol: Union[RequestProtocol, DEFAULT] = DEFAULT.VALUE,
```

Comment:
let's make this a private/undocumented option please! (prefix with `_`)

Reply:
actually even better would be to not put this in `.options()` and instead have a separate private method for setting it (`_set_request_protocol`). that way users aren't confused by it.